Dockerfile

```diff
@@ -1,28 +1,28 @@
-FROM openshift/origin-base
-
-ENV APP_HOME=/opt/app/src \
-    GOPATH=/go \
-    PATH="${PATH}:${GOPATH}/bin:/usr/local/bin" \
-    INSTALL_PKGS="golang make git"
+# first stage
+FROM openshift/origin-release:golang-1.10 AS builder
 
-RUN yum install -y ${INSTALL_PKGS} --setopt=tsflags=nodocs
+ENV GOPATH=/go
 
 WORKDIR /tmp/cluster-logging-operator
 COPY vendor/ /tmp/cluster-logging-operator/vendor
-COPY scripts/* /usr/local/bin/scripts/
 COPY cmd/ /tmp/cluster-logging-operator/cmd
 COPY pkg/ /tmp/cluster-logging-operator/pkg
 COPY Makefile /tmp/cluster-logging-operator
 
-RUN make && \
-    cp _output/bin/cluster-logging-operator /usr/local/bin/ && \
-    rm -rf /tmp/cluster-logging-operator && \
-    yum erase -y ${INSTALL_PKGS} && \
-    yum clean all
+RUN make
+
+# second stage
+
+FROM openshift/origin-base
+
+ENV PATH="${GOPATH}/bin:/usr/local/bin:${PATH}"
+
+COPY scripts/* /usr/local/bin/scripts/
+COPY --from=builder /tmp/cluster-logging-operator/_output/bin/cluster-logging-operator /usr/local/bin/
+
+ENTRYPOINT ["/usr/local/bin/cluster-logging-operator"]
```

Comment:
does it make a difference the layer order for `ENTRYPOINT`?

Reply:
I don't think it matters [1] but I can move to the bottom if you have a preference

[1] https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Reply:
should not matter